Modified Starter Sample file(c++ and java) version and fixed the bug

sasmita opened the pull request.

Propanu commented on kits/starter/c++/VariousSensors.cpp:
```diff
@@ -47,6 +8,11 @@
+#include <ssd1308.h>
+#include <ssd1306.h>
+#include <sainsmartks.h>
+#include <lcm1602.h>
```
Why were these added?

sasmita replied:
I see. I think, it got added mistakenly while linking other libraries. Please let me know, if you want me to submit a separate pull request.

From: Mihai Tudor Panu [mailto:notifications@github.com]
Sent: Monday, March 21, 2016 3:45 PM
To: intel-iot-devkit/iot-devkit-samples <iot-devkit-samples@noreply.github.com>
Cc: Patra, Sisinty S <sisinty.s.patra@intel.com>
Subject: Re: [iot-devkit-samples] Modified Starter Sample file(c++ and java) version and fixed the bug (#6)


In kits/starter/c++/VariousSensors.cpp<https://github.com/intel-iot-devkit/iot-devkit-samples/pull/6#discussion_r56910630>:

> @@ -47,6 +8,11 @@

>  #include <iomanip>

>  #include <string>

>  #include <sstream>

> +#include <ssd1327.h>

> +#include <ssd1308.h>

> +#include <ssd1306.h>

> +#include <sainsmartks.h>

> +#include <lcm1602.h>

Why were these added?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/intel-iot-devkit/iot-devkit-samples/pull/6/files/be09a2d6654b28aa713b722a4f8bf2b0b86dd28f#r56910630>

Propanu replied:
No need for a separate one, just force push the changes and this PR will reflect it. Thanks!

Propanu commented:
Fixed & merged. Thanks, but please be more thorough when formatting your patches!

sasmita commented:
Sure!!

From: Mihai Tudor Panu [mailto:notifications@github.com]
Sent: Wednesday, March 23, 2016 10:58 AM
To: intel-iot-devkit/iot-devkit-samples <iot-devkit-samples@noreply.github.com>
Cc: Patra, Sisinty S <sisinty.s.patra@intel.com>
Subject: Re: [iot-devkit-samples] Modified Starter Sample file(c++ and java) version and fixed the bug (#6)


Fixed & merged. Thanks, but please be more thorough when formatting your patches!

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/intel-iot-devkit/iot-devkit-samples/pull/6#issuecomment-200467054>
